src/components/ChromecastButton.ts

```diff
@@ -30,7 +30,7 @@ export class ChromecastButton extends StateReceiverMixin(CastButton, ['player'])
         const mask = this.shadowRoot!.querySelector<SVGClipPathElement>(`svg clipPath#${maskId}`)!;
         const rings = this.shadowRoot!.querySelector<SVGGElement>(`svg .theoplayer-chromecast-rings`)!;
         const uniqueMaskId = `${maskId}-${id}`;
-        mask.setAttribute('id', uniqueMaskId);
+        mask?.setAttribute('id', uniqueMaskId);
```

Comment:
This conflicts with the non-null assertion at the very end of this line: https://github.com/THEOplayer/web-ui/blob/252b96ffb306a28b9faf899125aa650501fa701f/src/components/ChromecastButton.ts#L30

The mask *should* always exist, right? So we shouldn't need to use `?.`...

Reply:
On WebOS 3.4 the mask was actually `null` causing a TypeError. Will take a deeper look at it.

Reply:
It's possible that these older browsers don't support `querySelector()` very well inside inline `<svg>` elements...

If that's the case, then we should the non-null assertion (`!`).